games-fps/eduke32/patches/eduke32-20180224_6694.patchset

```diff
@@ -1,40 +1,95 @@
-From a6cd111118bb90838cd5a4c02a0751717d610d45 Mon Sep 17 00:00:00 2001
+From 68865ee289078b991e257621597ef93b17887a86 Mon Sep 17 00:00:00 2001
 From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
-Date: Sat, 3 Jun 2017 16:02:01 +0200
-Subject: Haiku: endiannes fix
+Date: Sun, 25 Feb 2018 11:10:26 +0100
+Subject: [PATCH] Haiku supporting patches
 
+---
+ Common.mak                    | 20 ++++++++++++++------
+ GNUmakefile                   |  4 ++++
+ source/build/include/compat.h |  2 +-
+ source/build/src/kplib.cpp    | 20 ++++++++++++--------
+ source/duke3d/src/common.cpp  |  2 ++
+ 5 files changed, 33 insertions(+), 15 deletions(-)
 
+diff --git a/Common.mak b/Common.mak
+index f101b27..11fa9a6 100644
+--- a/Common.mak
++++ b/Common.mak
+@@ -33,6 +33,8 @@ ifndef HOSTPLATFORM
+             HOSTPLATFORM := DARWIN
+         else ifeq ($(findstring BeOS,$(uname)),BeOS)
+             HOSTPLATFORM := BEOS
++        else ifeq ($(findstring Haiku,$(uname)),Haiku)
++            HOSTPLATFORM=HAIKU
+         else ifeq ($(findstring skyos,$(uname)),skyos)
+             HOSTPLATFORM := SKYOS
+         else ifeq ($(findstring QNX,$(uname)),QNX)
+@@ -957,17 +959,23 @@ else ifeq ($(SUBPLATFORM),LINUX)
+     LIBS += -lrt
+ endif
+ 
+-ifeq (,$(filter $(PLATFORM),WINDOWS WII))
+-    ifneq ($(PLATFORM),BSD)
+-        LIBS += -ldl
+-    endif
+-    ifneq ($(PLATFORM),DARWIN)
+-        LIBS += -pthread
++ifneq ($(PLATFORM),HAIKU)
++    ifeq (,$(filter $(PLATFORM),WINDOWS WII))
++        ifneq ($(PLATFORM),BSD)
++            LIBS += -ldl
++        endif
++        ifneq ($(PLATFORM),DARWIN)
++            LIBS += -pthread
++        endif
+     endif
+ endif
+ 
+ LIBS += -lm
+ 
++ifeq ($(PLATFORM),HAIKU)
++    LIBS+= -lpthread -lnetwork
```

Comment:
`libpthread` is actually empty. I think it was there for some weird compatibility reasons.

```diff
-+    LIBS+= -lpthread -lnetwork
++    LIBS+= -lnetwork